src/components/dialog/dialog.js

```diff
@@ -548,27 +569,6 @@ function MdDialogProvider($$interimElementProvider) {
           return angular.element(closeButton);
         }
       }
-
```

Comment:
Did we replace this with some similar functionality so that the text would be wrapped with `<p>` tags? It looks like the paragraph tags are styled slightly differently than regular `<div>` contents, so this may cause some styling issues.